### ISSUE TITLE
configure issue

### ISSUE THREAD

**Issue by darrencl** [MRtrix3/mrtrix3#1952]
Hi,

I would like to install mrtrix3 in Windows machine during executing `./configure`, initially I had issue with QT5 library, which I fixed by appending `win32/` to `load` function in `windows-vulkan.conf`.

Currently, I am stuck with this following issue which I couldn't find someone else experiencing it. It seems that `qmake` looks for `Makefile.realease` in temporary directory which didn't exists for some reason. Please refer to configure.log for more detail. 

Version detail

```bash
$ qmake --version
QMake version 3.1
Using Qt version 5.14.1 in C:/msys64_new/mingw64/lib
```

Any help would be greatly appreciated.  

Thanks!

[configure.log](https://github.com/MRtrix3/mrtrix3/files/4202237/configure.log)

**Comment by jdtournier** [comment]
Thanks for reporting, I'll look into this when I have a minute. Sounds similar to [this issue on the forum](https://community.mrtrix.org/t/problem-with-windows-vulkan-sdk-when-installing-mrtrix3-on-windows/3300?u=jdtournier)? It's probably related to changes in the most recent version of Qt (I'm still using Qt 5.13). 

If you just want to get going in the meantime, one option is to use the (*much* smaller) Qt package I've made available recently - it's not advertised or documented yet, but will be soon for the next release. Brief instructions for using that:
```
pacman -R mingw-w64-x86_64-qt5
wget https://www.mrtrix.org/msys2/mingw-w64-x86_64-qt5-mrtrix-5.13.2-1-any.pkg.tar.xz
pacman -U mingw-w64-x86_64-qt5-mrtrix-5.13.2-1-any.pkg.tar.xz
```
Then try `./configure && ./build` again...

> initially I had issue with QT5 library, which I fixed by appending win32/ to load function in windows-vulkan.conf.

Can you provide more details about this? I've observed this recently, but it was just a a warning as far as I can tell. But if you have a fix, I'd like to hear more about it!

**Comment by jdtournier** [comment]
FIx is in #1954 - closing this issue.

**Comment by darrencl** [comment]
Thanks @jdtournier ! I'll try that.
